test/librato_tests.js

```diff
@@ -62,6 +62,22 @@ module.exports = {
     this.emitter.emit('flush', 123, metrics);
   },
 
+  testValidMeasurementSingleTagFromHistogram: function(test) {
+    test.expect(4);
+    let metrics = {gauges: {'my_gauge#foo=bar.bin_50': 1}};
```

Comment:
This is a good test but we probably want to build a proper `timer` metric to fully test this code path: https://github.com/librato/statsd-librato-backend/blob/master/lib/librato.js#L373.